test/trento_web/controllers/installation_controller_test.exs

```diff
@@ -0,0 +1,15 @@
+defmodule TrentoWeb.InstallationControllerTest do
+  use TrentoWeb.ConnCase, async: true
+
+  import OpenApiSpex.TestAssertions
+
+  alias TrentoWeb.OpenApi.ApiSpec
+
+  test "should return the api key", %{conn: conn} do
+    api_spec = ApiSpec.spec()
+
+    get(conn, Routes.installation_path(conn, :get_api_key))
```

Comment:
We agreed here that we are going to use "absolute" paths for testing. So this would be `api/installation` or whatever, without using the `Routes` option

Reply:
answered in the main comment